Block/Checkout/Success.php

```diff
@@ -58,6 +67,13 @@ public function isAllowInvalidateQuote()
         return (bool) (version_compare($this->getMagentoVersion(), '2.2.0', '<'));
     }
 
+    /**
+     * @return bool
+     */
+    public function shouldTrackCheckoutFunnel() {
+        return $this->configHelper->shouldTrackCheckoutFunnel();
```

Comment:
@daisy1754 I've just noticed Config::shouldTrackCheckoutFunnel reads / points to the same configuration path as Config::isActive, XML_PATH_ACTIVE instead of XML_PATH_TRACK_CHECKOUT_FUNNEL. Please update.

Reply:
good catch... will fix with separate PR